packages/core/src/entity/shipping-method/shipping-method.entity.ts

```diff
@@ -103,6 +102,7 @@ export class ShippingMethod
      * to prevent circular references which cause the JSON.stringify() to fail.
      */
     protected toJSON(): any {
-        return omit(this, ['allCheckers', 'allCalculators'] as any);
+        const { allCheckers, allCalculators, ...rest } = this;
```

Comment:
Does this behave differently than the existing solution? Are there any cases that this solves that the other does not?